sessions/recap-project-5/example-implementation/components/Header.js

```diff
@@ -0,0 +1,51 @@
+import styled from "styled-components";
+
+const StyledHeader = styled.header`
+  display: flex;
+  position: fixed;
+  flex-direction: column;
+  align-items: flex-end;
+  bottom: 0;
+  width: 100%;
+  max-width: 600px;
+  height: 20vh;
+  background: #000;
+  padding: 0 1rem;
+`;
+
+const Title = styled.h1`
+  font-size: 4rem;
+  color: #fff;
+  margin: 0;
+  border-bottom: 5px solid ${(props) => props.colorSet[0]};
+
+  &::before {
+    content: "!";
+    color: ${(props) => props.colorSet[0]};
+  }
+`;
+
+const HeaderText = styled.p`
+  color: #fff;
+`;
+
+const Spotlight = styled.p`
+  color: yellow;
+  align-self: flex-start;
+`;
+
+function Header({ randomImage }) {
+  const colorSet = randomImage.colors;
+
+  return (
+    <StyledHeader>
+      <Spotlight>
+        Spotlight: <small>©</small>ARTIST
+      </Spotlight>
+      <Title colorSet={colorSet}>ART</Title>
+      <HeaderText>Contemporary Art Gallery</HeaderText>
```

Comment:
I cannot see `HeaderText` and I see only parts of `Title` because there are overlapped from the navigation when I'm on the spotlight page (`/pages/index.js`).

I think this is because in `Layout.js`, the `Main` element has `height: 100vh` and the `Navigation` is rendered directly below the `Main`, but has `position: fixed` and `height: 5vh`?

Reply:
Wild. The height of main isn't the issue, thought. It's something to do with the height of the header and the `bottom` value. You can test it in your browser, too. :) I'll fix it.

Reply:
Fixed it @PReichetanz
Please pull the new changes so you can actually see the things you need to see haha